wetterdienst/provider/dwd/radar/access.py

```diff
@@ -212,6 +218,28 @@ def collect_radar_data(
                         if result.timestamp is None:
                             result.timestamp = date_time
 
+                        if fmt == DwdRadarDataFormat.BUFR:
+                            df = pdbufr.read_bufr(
```

Comment:
Do you have a strategy to test this ?

Reply:
At this moment I am only testing for the columns with `test_radar_request_site_historic_pe_bufr_dataframe` but it shouldn't be a big deal adding some unit test there as well.